components/telemetry-operator/controller/logpipeline/controller.go

```diff
@@ -252,13 +307,13 @@ func (r *Reconciler) updateMetrics(allPipelines *telemetryv1alpha1.LogPipelineLi
 type keepFunc func(*telemetryv1alpha1.LogPipeline) bool
 
 func count(pipelines *telemetryv1alpha1.LogPipelineList, keep keepFunc) int {
-	count := 0
+	c := 0
```

Comment:
Was this change intentional?

Reply:
I assume, GoLand was complaining since the variable name shadowed the func name, which was also count.